### DISCUSSION TITLE
the release -p option

### DISCUSSION THREAD

**Discussion by jftuga** [goreleaser/goreleaser#4808] [Q&A] [answered]
Is the `release -p` *(number of tasks to run concurrently)* enabled by default? In other words, if I do not include this switch, will `goreleaser` use all cores or just one? Otherwise, do I have to explicitly enable this behavior by including `-p`?

**Comment by caarlos0** [comment, marked as answer]
yes, it uses the number of CPUs by default

```
goreleaser release --help 
```

or https://goreleaser.com/cmd/goreleaser_release/
